### DISCUSSION TITLE
unable to pull custom image for airflow

### DISCUSSION THREAD

**Discussion by Sai05091990** [airflow-helm/charts#603] [Questions & Answers]
I am using values.yaml and passing pull secret and when i execute helm install command and pods are failing with error imagepullbackoff saying 403 error. 

i am use local gitlab container registry.

i can pull other images using the same pull secret but not with helm install.

**Comment by thesuperzapper** [comment]
@Sai05091990 all the images in the chart will use the single pull secret set by the `airflow.image.pullSecret` value.

Some questions:

1. What values are you using to set your pull secret?
2. What namespace is your pull-secret in? (it must be in the same namespace as your helm install)
